.circleci/config.yml

```diff
@@ -536,38 +536,38 @@ _parameters:
 
 workflows:
   version: 2
-  build_and_test:
```

Comment:
Should all the circle stuff just be removed?

Reply:
I think I'll migrate the CircleCI workflow too, so that we can still have both while we iron things out. But I think I'll have the GitHub action's checkout job trigger CircleCI for now, instead of CircleCI being directly triggered by GitHub events, so that CircleCI can use the same base image as the rest of the Github workflows.